addOns/automation/src/main/javahelp/org/zaproxy/addon/automation/resources/help/contents/tests.html

```diff
@@ -76,6 +76,31 @@ <H2>YAML</H2>
   
 </pre>
 
+<H2>Url Presence</H2>
+
+Presence tests are supported by the activeScan and passiveScan-wait jobs. These tests can be used to validate the
+presence/absence of specific expressions in the Http response/request. The expressions are specified in the YAML file as regular
+expressions. The test will pass if the expression is found in the response/request. Every test must have a mandatory url parameter
+which specifies the url to be tested, an operator parameter which specifies the operator to be used for the test and atleast one
+regex parameter which specifies the regular expression to be matched.
+<p>
+A job can have tests for multiple expressions, and multiple tests can be created for expressions having the same url.
+
+<H2>YAML</H2>
+<pre>
+  jobs:
+  - type: something
+    tests:
+      - name: 'test one'                      # Name of the test, optional
+        type: url                             # Specifies that the test is of type 'url'
+        url: http://www.example.com/path      # String: The url to be tested, mandatory
+        operator: 'and'                       # One of 'and', 'or', mandatory
```

Comment:
I think this should be optional with a default of "or" (but I can be convinced otherwise of the default value;)